hw/ip/csrng/data/csrng_testplan.hjson

```diff
@@ -13,55 +13,78 @@
     {
       name: smoke
       desc: '''
-            Verify sending instantiate/generate cmds via SW path.
-            Verify reading genbits via SW path.
+            Verify that sending an instantiate command via the SW path returns OK.
+            Instantiate the CSRNG with flag0 set to true and clen set to 12.
+            Verify that sending a generate command via the SW path returns glen=1 number of words
+            followed by an OK.
             '''
       stage: V1
       tests: ["csrng_smoke"]
     }
     {
       name: interrupts
       desc: '''
-            Verify cs_cmd_req_done interrupt asserts/clears as predicted.
-            Verify cs_entropy_req interrupt asserts/clears as predicted.
-            Verify cs_hw_inst_exc interrupt asserts/clears as predicted.
-            Verify cs_fifo_err interrupt asserts/clears as predicted.
-            Verify fifo error status bits are set as predicted.
+            This test verifies the behavior of the Interrupt State Register.
+            Verify cs_cmd_req_done interrupt asserts when glen number of genbit words have been
+            generated.
+            Verify cs_entropy_req interrupt asserts when instantiate or reseed is called with flag0
+            set to false.
+            Verify cs_hw_inst_exc interrupt asserts when generating bits and the reseed counter
+            exceeds the reseed interval of 2^48.
+            Verify cs_fatal_err interrupt asserts when any bit of err_code register is set.
+            Verify that each interrupt clears when the next command is received.
             '''
       stage: V2
       tests: ["csrng_intr"]
     }
     {
       name: alerts
       desc: '''
-            Verify recov_alert asserts as predicted.
-            Verify all recov_alert_sts bits assert/clear as predicted.
-            Verify fatal_alert asserts as predicted.
+            Verify recov_alert asserts when recov_alert_sts becomes non-zero.
+            Verify each respective bit of recov_alert_sts asserts when:
+            - Either of the 3 values in the control register are not valid kMultiBitBool values.
+            - When an initiate or reseed command is sent where flag0 is not a valid kMultiBitBool
+            value.
```

Comment:
Invalid MuBi values on flag0 is currently untested (at least I didn't see it). We need a TODO for this. Alternatively, we should add an explicit cover point unter `csrng_cmds_cg` for this.

Reply:
Added a TODO in the testplan for this.